docs/topics/feed-exports.rst

```diff
@@ -347,6 +370,19 @@ as a fallback value if that key is not provided for a specific feed definition:
 
 -   ``fields``: falls back to :setting:`FEED_EXPORT_FIELDS`.
 
+-   ``item_classes``: list of acceptable :ref:`item classes <topics-items>` that should be exported to the feed storage.
+
+    If no items declared then there will be no filtering based on item_classes. There is no fallback value.
+
+    .. versionadded:: 2.X.X
```

Comment:
I am not sure which version to write here or if this was needed at all.

Reply:
See https://docs.scrapy.org/en/latest/contributing.html#documentation-policies

Reply:
So as this is a new feature, we should include it in 2.6.0 version release?

Reply:
My point is that you can use `VERSION` as version. Whenever a release includes this change, be it 2.6 or a later version, we will replace `VERSION` with the actual version.